google/cloud/security/common/gcp_api/cloud_resource_manager.py

```diff
@@ -96,14 +95,13 @@ def get_projects(self, resource_name, **filterargs):
 
         # TODO: Convert this over to _base_client._build_paged_result().
```

Comment:
The paged result is a separate change (and a separate line item in the workstream), so will tackle that next.